test/AutobahnTestApp/AutobahnTestApp.csproj

```diff
@@ -1,16 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  
```

Comment:
VS no longer auto-deletes empty lines. Separate blocks per group is more readable.